godot-core/src/builtin/collections/packed_array.rs

```diff
@@ -378,6 +378,29 @@ macro_rules! impl_packed_array {
             pub fn as_inner(&self) -> inner::$Inner<'_> {
                 inner::$Inner::from_outer(self)
             }
+
+            /// Create array filled with default elements.
+            fn default_with_size(n: usize) -> Self {
+                let mut array = Self::new();
+                array.resize(n);
+                array
+            }
+
+            /// Moves elements from slice.
```

Comment:
```suggestion
            /// Moves elements from slice.
```
Comments should express more than the function name, or be omitted.

Here, you could mention something like
```suggestion
            /// Drops all elements in self and replaces them with ones in provided pointer range.
```